.php_cs

```diff
@@ -0,0 +1,39 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create();
+
+return PhpCsFixer\Config::create()
+    ->setRules([
+        '@PSR2'                                 => true,
+        '@DoctrineAnnotation'                   => true,
+        '@Symfony'                              => true,
+        'align_multiline_comment'               => ['comment_type' => 'all_multiline'],
+        'array_syntax'                          => ['syntax' => 'short'],
+        'binary_operator_spaces'                => ['default' => 'align'],
+        'blank_line_after_opening_tag'          => true,
+        'cast_spaces'                           => ['space' => 'single'],
+        'compact_nullable_typehint'             => true,
+        'concat_space'                          => ['spacing' => 'one'],
+        'declare_strict_types'                  => true,
+        'dir_constant'                          => true,
+        'increment_style'                       => ['style' => 'post'],
```

Comment:
`$++i` and $i++` are meaningfully different things. This shouldn't be in a CS change.